llvm/docs/ReleaseNotes.rst

```diff
@@ -130,6 +132,8 @@ Changes to the Hexagon Backend
 
 Changes to the LoongArch Backend
 --------------------------------
+* Respect the code model attribute of global variables when different from
+  their default values.
```

Comment:
"Code model of global variables can now be overridden by means of the newly-added `code_model` LLVM IR attribute."?